runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java

```diff
@@ -296,4 +314,32 @@ private HiscoreEndpoint getLocalHiscoreEndpointType()
 		}
 		return HiscoreEndpoint.NORMAL;
 	}
+
+	private HiscoreEndpoint getHiscoreEndpointFromName(String rawName)
+	{
+		// Remove <col> tags, as they come before the <img> tags we need.
+		Matcher macther = COL_STRIPPER.matcher(rawName);
+		if (macther.matches())
+		{
+			rawName = macther.group(1);
+		}
+
+		if (!rawName.startsWith("<"))
+		{
+			return HiscoreEndpoint.NORMAL;
+		}
+		else if (rawName.startsWith(IconID.IRONMAN.toString()))
+		{
+			return HiscoreEndpoint.IRONMAN;
+		}
+		else if (rawName.startsWith(IconID.ULTIMATE_IRONMAN.toString()))
+		{
+			return HiscoreEndpoint.ULTIMATE_IRONMAN;
+		}
+		else if (rawName.startsWith(IconID.HARDCORE_IRONMAN.toString()))
+		{
+			return HiscoreEndpoint.HARDCORE_IRONMAN;
+		}
+		return null;
```

Comment:
I see no reason why this should ever be able to default to anything other than `NORMAL`